README.md

```diff
@@ -1,2 +1,46 @@
 # beer-is-poetic
 Beer is Poetic
+
```

Comment:
If I remember correctly, we need to put the repository and live site links in the README